xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/pagesuggest.vm

```diff
@@ -0,0 +1,85 @@
+#macro (suggestPages)
```

Comment:
I wouldn't add this new template unless there are good reasons that prevent us from using the existing REST API. See my comment above.